src/components/StyleGuide/ColorRow/ColorRow.js

```diff
@@ -7,7 +7,7 @@ const classnames = require('classnames/bind').bind(style);
 const ColorRow = props => (
   <div className={style.row}>
     {props.colors.map(color => (
-      <div className={style.swatchWrapper}>
+      <div key={color.variableName} className={style.swatchWrapper}>
```

Comment:
Unrelated. I don't know how, but I didn't have keys for any of these and never noticed the errors.